pymc_marketing/mmm/base.py

```diff
@@ -1146,6 +1146,8 @@ def compute_mean_contributions_over_time(
                 .to_dataframe()
                 .squeeze()
                 .unstack()
+                .sum(axis=1),
+                columns=["yearly_seasonality"],
```

Comment:
How will someone know that `yearly_seasonality` is available in the `plot_grouped_contribution_breakdown_over_time`? 

outside scope of PR: But would it make sense to store this as a deterministic variable in our model?

Reply:
> How will someone know that yearly_seasonality is available in the plot_grouped_contribution_breakdown_over_time?

Should be describe in the example.


> outside scope of PR: But would it make sense to store this as a deterministic variable in our model?

I have been thinking on it. I'll make it in another PR

Reply:
Done!